crates/orderbook/src/run.rs

```diff
@@ -437,7 +437,11 @@ pub async fn run(args: Arguments) {
                 )
                 .unwrap(),
             },
-        ))
+        );
+        match args.enforce_verified_quotes {
+            true => Arc::new(quoter.enforce_verification(balance_fetcher.clone())),
+            false => Arc::new(quoter),
+        }
     };
     let optimal_quoter = create_quoter(price_estimator.clone());
     let fast_quoter = create_quoter(fast_price_estimator.clone());
```

Comment:
```suggestion
    let optimal_quoter = create_quoter(price_estimator);
    let fast_quoter = create_quoter(fast_price_estimator);
```

Reply:
Those objects are already `Arc`ed but indeed let's not `.clone()` completely unnecessarily.